### ISSUE TITLE
WCAG 2.2 Violation - There are changes in status in the interface that are not announced to users.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#46]
#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- [4.1.3 Status Messages](https://www.w3.org/TR/WCAG22/#status-messages)

#### Further information:

1. 

### Instances

1. #47 
2. #48
